ui/components/app/permissions-connect-permission-list/index.scss

```diff
@@ -0,0 +1,22 @@
+.permissions-connect-permission-list {
+  .permission {
+    @include H6;
+
+    width: 100%;
```

Comment:
Is this needed? Should be 100% by default?

Reply:
It's needed. I tried multiple things, haven't found any that works well